bazel/envoy_library.bzl

```diff
@@ -80,13 +136,24 @@ EXTENSION_STATUS_VALUES = [
 def envoy_cc_extension(
         name,
         security_posture,
+        category = None,
         # Only set this for internal, undocumented extensions.
         undocumented = False,
         status = "stable",
         tags = [],
         extra_visibility = [],
         visibility = EXTENSION_CONFIG_VISIBILITY,
         **kwargs):
+    if not category:
+        print("FAILCAT %s not set" % name)
+        # fail("Category not set for %s" % name)
```

Comment:
Let's fail already :)

Reply:
it was failing and i was struggling to get the error - apologies for last minute hackery